src/views/Home/components/FAQ/index.scss

```diff
@@ -0,0 +1,81 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.faq-component {
+  margin-top: -1px;
+  background-size: cover;
+  background: linear-gradient(
+    var(--color-light-pink),
+    var(--color-lavender),
+    var(--color-sky-blue)
+  );
+  &__box {
+    margin: auto;
+    border: 7px solid rgba(255, 255, 255, 0.75);
+    border-radius: 59px;
+    padding: 10px;
+    display: table;
+    // height: 1400px;
+    width: 80%;
+    @include responsiveness.screen(smallest, phone, desktop, big-desktop) {
+      width: 60%;
+    }
```

Comment:
The width + font-size still needs to be refined a little for galaxy fold. So I would update this in general for `responsiveness.screen(smallest)`.

<img width="348" alt="faqSmallest" src="https://user-images.githubusercontent.com/38510322/140803219-c25219ec-586f-4c4e-9248-510e7a05eed2.PNG">